hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,8 @@
+RELEASE_TYPE: minor
+
+:func:`~hypothesis.extra.django.from_field` now supports
+:class:`~django.contrib.auth.forms.UsernameField` and
+:class:`~django.contrib.auth.forms.ReadOnlyPasswordHashField`
+from ``django.contrib.auth.forms``.
+
+Thanks to Afonso Silva for reporting and fixing :issue:`3417`.
```

Comment:
```suggestion
:class:`~django.contrib.auth.forms.UsernameField` from ``django.contrib.auth.forms``.

Thanks to Afonso Silva for reporting and working on :issue:`3417`.
```

Since we already supported the other field, and have some remaining things in the issue.